modules/styling/stories/mdx/Overview.mdx

```diff
@@ -397,6 +397,6 @@ const ThemedCard = ({isDarkTheme, headerColor, elemProps}) => {
   <InformationHighlight.Icon />
   <InformationHighlight.Heading>Information</InformationHighlight.Heading>
   <InformationHighlight.Body>
-    For a more in depth overview, please view our <Hyperlink href="https://workday.github.io/canvas-kit/?path=/docs/styling-getting-started-stencils--docs">Create Stencil</Hyperlink> docs.
+    For a more in depth overview, please view our <Hyperlink href="https://workday.github.io/canvas-kit/?path=/docs/styling-getting-started-create-stencil--docs">Create Stencil</Hyperlink> docs.
```

Comment:
Fixed the Create Stencil link.

I tried updating the `routes.js` file, but that turned out to be a hassle (`routes.js` only rewrites links in the `[link](url)` Markdown format, and the site doesn't render `InformationHighlight` properly if we use that format). I stuck with the `workday.github.io` links.